packages/core/admin/admin/src/pages/Marketplace/hooks/useMarketplaceData.ts

```diff
@@ -14,6 +14,7 @@ interface UseMarketplaceDataParams {
   debouncedSearch: string;
   query?: MarketplacePageQuery;
   tabQuery: TabQuery;
+  strapiVersion?: string | null;
```

Comment:
Should it be optional? I think in v5 onwards we can expect it to be there every time?

Reply:
yeah I wasn't really sure about that, the `strapiVersion` originally coming from the `AppInfo` context where it was typed like this

Reply:
Ah okay, makes sense if it matches the types there